robot_ws/src/cloudwatch_robot/setup.py

```diff
@@ -1,12 +1,32 @@
+import os
+from shutil import copyfile
+
+from ament_index_python.packages import get_package_share_directory
 from setuptools import find_packages, setup
+from setuptools.command.install import install
+
 
 package_name = 'cloudwatch_robot'
 
+
+class CopyRvizModelToSrc(install):
+
+    def run(self):
+        src = get_package_share_directory('turtlebot3_navigation2')+'/rviz/tb3_navigation2.rviz'
```

Comment:
I think there is some naming confusion here; `src` could mean too many different things now. Let's rename `CopyRvizModelToSrc` to like `CopyRvizModelToPackageDir` or something and rename `src` to `rviz_file` or something.